api/src/main/java/org/pmiops/workbench/actionaudit/auditors/UserServiceAuditAdapterImpl.java

```diff
@@ -76,4 +76,21 @@ public void fireAdministrativeBypassTime(
 
     actionAuditService.send(eventBuilder.build());
   }
+
+  @Override
+  public void fireAcknowledgeTermsOfService(DbUser targetUser, Integer termsOfServiceVersion) {
+    actionAuditService.send(
+        ActionAuditEvent.builder()
+            .timestamp(clock.millis())
+            .agentType(AgentType.USER)
+            .agentId(dbUserProvider.get().getUserId())
```

Comment:
Please add a comment about why you use the provider here and the passed-in user below if that's intentional.